users/routers/routerAddresses.ts

```diff
@@ -2676,5 +2676,15 @@ export default ([
             ]
         }
     }, 
+    {
+        "id":"3388", // todo?
```

Comment:
why is the id set to 3388? drift's id is 970

the protocol with id 3388 is sanctum